src/libraries/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngProvider.cs

```diff
@@ -86,6 +86,14 @@ public override string ToString()
         // Well known NCrypt KSPs
         //
 
+        public static CngProvider MicrosoftPlatformCryptoProvider
+        {
+            get
+            {
+                return s_msPlatformKsp ?? (s_msPlatformKsp = new CngProvider("Microsoft Platform Crypto Provider")); // MS_PLATFORM_CRYPTO_PROVIDER
+            }
+        }
+
```

Comment:
@bartonjs, are there tests somewhere for the existing statics here?  I did a quick search but couldn't find any.